src/features/emojiMod.ts

```diff
@@ -69,6 +71,43 @@ export const reactionHandlers: ReactionHandlers = {
 
     reportUser({ reason, message, staff, members });
   },
+  "🔍": async ({ message, usersWhoReacted }) => {
+    const NUMBER_OF_REACTIONS_REQUIRED_FOR_DELETION = 2;
+
+    const hasRequiredNumberOfReactions =
+      usersWhoReacted.length >= NUMBER_OF_REACTIONS_REQUIRED_FOR_DELETION;
+
+    const hasAuthoritativeUserReacted = usersWhoReacted.some(
+      (userWhoReacted) => isStaff(userWhoReacted) || isHelpful(userWhoReacted),
+    );
+
+    if (!hasRequiredNumberOfReactions || !hasAuthoritativeUserReacted) {
+      return;
+    }
+
+    const newThread = await message.startThread({
+      name: createNewThreadName({ username: message.author.username }),
+    });
+
+    await newThread.send({
+      embeds: [
+        {
+          title: "Please improve your question",
+          type: "rich",
+          description: `
+            Sorry, our most active helpers have flagged this as a question that needs more work before a good answer can be given. This may be because it's ambiguous, too broad, or otherwise challenging to answer. 
+
```

Comment:
Could we consider adding a line that we prefer questions with !code attached, instead of screenshots or photos of code?

Reply:
Yeah, I think that'd be worthwhile 👍 do you have a suggestion or shall we take an excerpt from the !code command?

Reply:
Maybe have the bot send another message with the !code command? Double down on the spam?

Reply:
I'm not actually sure if the bot can trigger itself, I'd have to have a look at that. I'll try it and go from there

Reply:
Maybe it would be good to link to a set of Discord tips? Maybe a new page on reactiflux.com, perhaps?

Reply:
Yeah, that's a nice idea 👍  perhaps we can look at doing that in a second pass as I'd like to release this feature as soon as possible. What do you think?

Reply:
The bot can't trigger itself and I'm not sure it makes sense to start pulling the command outputs out so they can be reused at this point. I like Carl's suggestion and I'd be happy to release this as-is and iterate on it in the future when we've had time to build a new page on the website. What do you think?

Reply:
Into deferring a new page for late 👍